### PR TITLE
Added overwrite and uninstall options to unify and check for argument errors

### PR DESCRIPTION
**Pull request by verdx**
|Related issue|
|---|
| #1109 |

## Description

- Adds an uninstall option to remove all packages and files related to the stack
- Adds an overwrite option to remove all packages that you are trying to install
- You now have to pass the kibana node name when using argument `-k|--kibana`
- Adds checks to see if the arguments are correct and can go together
- Checks if any of the components is installed and gives an error if you try to install any of them
- Checks if certificates has been already created
- Checks if node names of the different components are correct, different and the same as in config.yml
- Checks if there are certificates for the nodes being installed

### CODE REVIEW

**Comment by verdx** [comment]
The overwrite and unattended options work perfectly, but it has been difficult integrating them with the progress bar. I have been trying different things but can't seem to get it to work.

File: unattended_scripts/install_functions/opendistro/kibana.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 installKibana() {
     
-    logger "Installing Open Distro for Kibana."
+    logger "Starting the Kibana installation."
```
Comment (lastranikos) [inline unattended_scripts/install_functions/opendistro/kibana.sh:10]: This should be so? or are we already changing the name to Wazuh Dashboard?

File: unattended_scripts/install_functions/opendistro/elasticsearch.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 installElasticsearch() {
 
-    logger "Installing Open Distro for Elasticsearch."
+    logger "Starting the Elasticsearch Installation"
```
Comment (lastranikos) [inline unattended_scripts/install_functions/opendistro/elasticsearch.sh:10]: Same here.
This should be so? or are we already changing the name to Wazuh indexer??

**Review by lastranikos — CHANGES_REQUESTED** [review]
LGTM
I only have the couple of doubts that I add below.

File: unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh [hunk 3]
```diff
@@ -292,27 +296,36 @@ main() {
...
+        mkdir ${base_path}/certs
+    fi
+
+
 
     if [ -n "$1" ]; then
         while [ -n "$1" ]
         do
-            case "$1" in
+            case "$1" in
```
Comment (DFolchA) [inline unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh:312]: Please remove trailing spaces

**Review by DFolchA — CHANGES_REQUESTED** [review]

**Review by DFolchA — APPROVED** [review]
LGTM

File: unattended_scripts/install_functions/opendistro/kibana.sh (hunk 1 above)
Comment (verdx) [reply unattended_scripts/install_functions/opendistro/kibana.sh:10]: This should be so, yes. We are first finishing the unification of the installers and when that is finished we will upgrade the names according to the new version.

**Review by DFolchA — APPROVED** [review]
LGTM

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**